data/pools/transactionPool.go

```diff
@@ -72,6 +76,7 @@ func MakeTransactionPool(ledger *ledger.Ledger, transactionPoolStatusSize int, l
 		ledger:          ledger,
 		statusCache:     makeStatusCache(transactionPoolStatusSize),
```

Comment:
The `statusCache` contains only the *failed* transaction groups. Shouldn't we rename it ?

Reply:
Yeah, it is that.
```
// failedTxnStatuses records failure messages from recently failed transactions, indexed by Txid.
// This allows algod clients to find out why the transaction they submitted failed.
failedTxnStatuses: makeStatusCache(...)
```